pandas/plotting/_core.py

```diff
@@ -796,14 +793,12 @@ def __call__(self, *args, **kwargs):
             if isinstance(data, ABCDataFrame):
                 return plot_backend.plot(data, x=x, y=y, kind=kind, **kwargs)
             else:
-                raise ValueError(
-                    ("plot kind {} can only be used for data frames").format(kind)
-                )
+                raise ValueError(f"plot kind {kind} can only be used for data frames")
         elif kind in self._series_kinds:
             if isinstance(data, ABCDataFrame):
                 if y is None and kwargs.get("subplots") is False:
-                    msg = "{} requires either y column or 'subplots=True'"
-                    raise ValueError(msg.format(kind))
+                    msg = f"{kind} requires either y column or 'subplots=True'"
+                    raise ValueError(msg)
```

Comment:
No need for an intermediate ``msg`` here, just put the message inside ValueError.